x/pairing/types/badge.go

```diff
@@ -1,9 +1,16 @@
 package types
 
 import (
+	"strconv"
+
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+type BadgeData struct {
```

Comment:
move this struct to the function of relaypayment as it is used only there

Reply:
done